dask/dataframe/tests/test_io.py

```diff
@@ -757,11 +757,17 @@ def test_to_hdf_multiple_datasets():
         out = pd.read_hdf(os.path.join(dn, 'data_aa.h5'), '/data')
         tm.assert_frame_equal(out, df.iloc[2:])
 
-    # saving to different datasets in multiple files with custom name_function
+    # triggering too many asterisks error
     with tmpdir() as dn:
         with pytest.raises(ValueError):
             fn = os.path.join(dn, 'data_*.h5')
-            a.to_hdf(fn, '/data_*', name_function=lambda i: 'a' * (i +  1))
+            a.to_hdf(fn, '/data_*')
```

Comment:
Not really, just saw it from a previous PR of mine and thought it'd be wasteful to change in it's own pr.
It might be a better fit in my `to_hdf` PR, but cba.